amplify/backend/function/risestandstrongcb34053a/src/utils/aws-utils.js

```diff
@@ -58,6 +58,44 @@ async function getAnnouncements(){
     }
 }
 
+/**
+ * Put updated announcement into table in DynamoDB.
+ * Returns nothing.
+ */
+async function putAnnouncement(data){
```

Comment:
You should pass in the announcementId separate from data, it makes it easier to modify things in the future